src/entries/popup/pages/settings/rpcs.tsx

```diff
@@ -254,7 +254,8 @@ export function SettingsNetworksRPCs() {
                   <ContextMenu>
                     <ContextMenuTrigger
                       disabled={
-                        mainnetChains[index].name === supportedChain?.name
+                        supportedChain?.rpcUrls.default.http[0] ===
+                        chain.rpcUrls.default.http[0]
```

Comment:
the logic we had here was disabling the option to remove the rpc url of rainbow supported chains, now we'll only block the removal of the rainbow rpc